css/style.css

```diff
@@ -63,22 +66,167 @@ body {
 #container {
   width: 100%;
   margin: auto;
-  height: 500px;
+
 
 }
 
 header {
+font-weight: 700;
+font-size: 24px;
+color: #ffffff;
+padding-top: 10px;
+padding-left: 80px;
+}
+
+h2 {
+  font-weight: 500;
+  font-size: 24px;
+  color: #ffffff;
+  text-align: center;
+  padding-top: 45px;
+  padding-bottom: 35px;
+  margin: auto;
+
+
+}
+
+.submit-button {
+  font-weight:800;
+  font-size: 14px;
+  border-radius: 5px 5px 5px 5px;
+  background-color: #f9e42e;
+  padding: 8px 35px;
+  border: none;
+  font-family: 'Open Sans', sans-serif;
+  margin: auto;
+  color: #033048;
+} 
+
+
+
+.border_box {
```

Comment:
try to be consistent with how you name your CSS classes. use either `-` to separate words or `_` but try to not mix using both